### PR TITLE
web: move colors out from `.theme-redesign`

### PR DESCRIPTION
**Pull request by valerybugakov**
## Changes

- Moved colors out from `.theme-redesign`.
- Replaced colors not present in [our design system](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=908%3A7961) with the closest alternatives based on [our Figma archives](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=3222%3A44203).

Part of https://github.com/sourcegraph/sourcegraph/issues/20847.
Closes https://github.com/sourcegraph/sourcegraph/issues/23771.

### CODE REVIEW

File: client/branded/src/components/ToggleBig.scss [hunk 1]
```diff
@@ -86,12 +86,8 @@ $toggle-big-height: 2rem;
         opacity: 0.2;
     }
 
-    &__icon {
-        fill: $secondary-light;
```
Comment (eseliger) [inline client/branded/src/components/ToggleBig.scss:90]: is this one not required anymore?

**Review by eseliger — APPROVED** [review]
🧹

File: client/branded/src/components/ToggleBig.scss (hunk 1 above)
Comment (valerybugakov) [reply client/branded/src/components/ToggleBig.scss:90]: Yep, according to the markup from the component: 

https://github.com/sourcegraph/sourcegraph/blob/24e2b4e9262b350d02de5db903c20fa97f822255/client/branded/src/components/ToggleBig.tsx#L70-L105

**Comment by umpox** [comment]
Sooooo I totally forgot to get around to merging this PR: https://github.com/sourcegraph/sourcegraph/pull/22096 😬 

Question: Did you follow the same approach?

The main thing is that all of the deprecated values are mapped over correctly following the FIgma spec: https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=3222%3A44203. The Figma link is this PR doesn't seem to load anything specific for me

**Comment by valerybugakov** [comment]
@umpox oh, lol, I forgot about this PR. 😬 It's been a long time.
Yeah, I used the [same diagram](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=3222%3A44203) for color migrations as you linked. Updated link in the PR description.

**Review by vovakulikov — APPROVED** [review]
Now there are fewer sass variables. 👍

**Pull request merged**